src/Controls/src/Core/VisualElement/VisualElement.cs

```diff
@@ -285,7 +285,10 @@ static void OnTransformChanged(BindableObject bindable, object oldValue, object
 			propertyChanged: (bindable, oldvalue, newvalue) =>
 			{
 				if (newvalue != null)
+				{
 					(bindable as VisualElement)?.NotifyBackgroundChanges();
+					((Brush)newvalue).Parent = bindable as VisualElement;
```

Comment:
We are going to wait for the build and tests, but I think this will cause a memory leak.
cc @jonathanpeppers

Reply:
Should something here at minimum _unset_ `Parent`?

Reply:
@jonathanpeppers I added an unsetting of a parent. Please let me know if it makes sense in this place.